framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -601,7 +634,36 @@ public abstract class SourceChecker extends AbstractTypeProcessor implements Opt
   private boolean showPrefixInWarningMessages;
 
   /** True if the -AwarnUnneededSuppressions command-line argument was passed. */
-  private boolean warnUnneededSuppressions;
+  boolean warnUnneededSuppressions;
+
+  /**
+   * The full list of subcheckers that need to be run prior to this one, in the order they need to
+   * be run in. This list will only be non-empty for the one checker that runs all other
```

Comment:
It can only be null or an immutable list which is empty for all but the ultimate parent checker. (I updated the documentation.) Can you point out where it can be a mutable list?